modules/on_gke/main.tf

```diff
@@ -0,0 +1,164 @@
+/**
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+//*****************************************
+//  Create Keys for Forseti server and
+//  client service accounts.
+//*****************************************
+
+resource "google_service_account_key" "server_key" {
+  service_account_id = "${var.forseti_server_service_account}"
+}
+
+resource "google_service_account_key" "client_key" {
+  service_account_id = "${var.forseti_client_service_account}"
+}
+
+//*****************************************
```

Comment:
Should change to "Grant IAM role to GKE service account" or something along those lines.

Reply:
@hshin-g I apologize, not sure I follow.

Reply:
The docstring for this is: `Create Keys for Forseti server and client service accounts.` which is the same docstring as the one above it. It should either be taken out (since it is redundant) or changed to reflect what the next resource(s) being created is/are.

Reply:
ok, thanks for the clarification